src/app/carbon-estimation/carbon-estimation.component.ts

```diff
@@ -1,125 +1,79 @@
-import { ChangeDetectorRef, Component, ElementRef, OnDestroy, OnInit, ViewChild, computed, input } from '@angular/core';
-import { CarbonEstimation } from '../types/carbon-estimator';
-import { NumberObject, sumValues } from '../utils/number-object';
-import { ApexAxisChartSeries, ChartComponent, NgApexchartsModule } from 'ng-apexcharts';
-
-import { startCase } from 'lodash-es';
-import {
-  EmissionsColours,
-  EmissionsLabels,
-  SVG,
-  getBaseChartOptions,
-  estimatorHeights,
-  tooltipFormatter,
-  placeholderData,
-  ApexChartSeriesItem,
-  ApexChartDataItem,
-} from './carbon-estimation.constants';
+import { ChangeDetectorRef, Component, computed, ElementRef, input, OnDestroy, OnInit, ViewChild } from '@angular/core';
 import { ExpansionPanelComponent } from '../expansion-panel/expansion-panel.component';
-import { Subscription, debounceTime, fromEvent } from 'rxjs';
+import { TabsComponent } from '../tab/tabs/tabs.component';
+import { TabItemComponent } from '../tab/tab-item/tab-item.component';
+import { CarbonEstimationTreemapComponent } from '../carbon-estimation-treemap/carbon-estimation-treemap.component';
+import { CarbonEstimation } from '../types/carbon-estimator';
+import { sumValues } from '../utils/number-object';
+import { estimatorHeights } from './carbon-estimation.constants';
+import { debounceTime, fromEvent, Observable, Subscription } from 'rxjs';
```

Comment:
```suggestion
import { debounceTime, fromEvent, Subscription } from 'rxjs';
```
I don't think `Observable` is being used.